### PR TITLE
Adding FAQ-s on a main elements for topic list and topic browser.

### PR DESCRIPTION
**Pull request by michael200kg**
Adding FAQ-s on a main elements for topic list and topic browser.

### CODE REVIEW

File: ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                <b>After the first fetch</b>, the list of columns will be populated with all the fetched columns and headers..
+            </p>
+        </li>
+        <li>
+            <p>You can filter by key, payload field, and message header.</p>
+        </li>
+
+        <li>
+            <p>
+                The filter supports various matching operations, including <b>"equals"</b>, <b>"contains"</b>, <b>"startWith"</b>, <b>"like"</b> and <b>"not empty"</b> operations.
```
Comment (YevgeniyYP) [inline ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html:17]: the last 'operations' is redundant here

File: ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        <li>
+            <p>
+                All chosen filters are combined using the <b>AND</b> operator.
+            </p>
+        </li>
+
+        <li>
+            <p>
+
+                If the chosen column has <b>JSON</b> or <b>Map</b> format,
```
Comment (YevgeniyYP) [inline ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html:31]: has -> is of ...

File: ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                including nested arrays, you can use dot notation to reference the nested parts.<br/>
+                For example, "a.b" on the left side of the filter references {a: {b: 123, c: "example"}},
+                or {a: [{b: 123, c: "example"},{b:100, c: "example2"}]}.
+                The supported matching operations for these formats are <b>"equals"</b>, <b>"contains"</b>, <b>"startWith"</b>, <b>"like"</b>.
```
Comment (YevgeniyYP) [inline ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html:35]: ... and <b>"like"</b>

File: ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                All chosen filters are combined using the <b>AND</b> operator.
+            </p>
+        </li>
+
+        <li>
+            <p>
+
+                If the chosen column has <b>JSON</b> or <b>Map</b> format,
+                including nested arrays, you can use dot notation to reference the nested parts.<br/>
+                For example, "a.b" on the left side of the filter references {a: {b: 123, c: "example"}},
```
Comment (YevgeniyYP) [inline ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html:33]: The example is not clear to me

File: ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+                The filter operates only on a set of records within a given timeout.
+                The offsets of these records are indicated in the offset range table.
+                If the set of records does not contain the desired data and the end offset has not been reached yet,
+                simply press the <img src="/kafka-manager/faq/topic-browser/img/fetch-forward-button.png"> button to search in
+                records with higher offsets.
+            </p>
+        </li>
+
+        <li>
+            <p>The filter also has the ability to filter records based on <b>header values</b>.</p>
```
Comment (YevgeniyYP) [inline ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html:50]: There is the ability to filter record....

File: ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+            <p>To apply filter across headers choose <b>"header: header-name"</b> in a column list and use <b>"headerValue"</b> or
+                <b>"subHeaderKey: subHeaderValue"</b> notation.</p>
```
Comment (YevgeniyYP) [inline ui/src/main/resources/static/faq/topic-browser/browse-filter-faq.html:52]: It is not clear, what the **subHeaderKey** exactly meant for. Maybe couple of examples would useful?

File: ui/src/main/resources/static/faq/topic-browser/fetch-faq.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        </li>
+        <li>
+            <p>
+                Each <b>Fetch</b> operation retrieves a specified number of records called the "Limit"
+                <b>(the limit can be set in the <img src="/kafka-manager/faq/topic-browser/img/limit-input-item.png"/> input item)</b>.
+                If the topic has multiple partitions, KM automatically calculates the limit for each partition.
+                The sum of all partition limits equals the total limit you entered. The limits for each partition are <b>distributed evenly</b>.
+                For example, if the "Limit" is set to 100 and the topic has 16 partitions, each partition will have a limit of 6 records.
+                Therefore, after a fetch operation, KM will attempt to fetch 6 new records from each partition.
+                If a specific partition does not have 6 records, all the remaining records in that partition will be fetched,
```
Comment (YevgeniyYP) [inline ui/src/main/resources/static/faq/topic-browser/fetch-faq.html:20]: It is not clear to me... If a partition does not have 6 record, but 4 only, where KM would fetch rest of the 2 records from?

**Review by YevgeniyYP — CHANGES_REQUESTED** [review]

File: ui/src/main/resources/static/faq/topic-browser/fetch-faq.html (hunk 7 above)
Comment (michael200kg) [reply ui/src/main/resources/static/faq/topic-browser/fetch-faq.html:20]: "all the remaining records in that partition will be fetched" that means that only 4 records will be fetched.

Comment (YevgeniyYP) [reply ui/src/main/resources/static/faq/topic-browser/fetch-faq.html:20]: Aaa... I got it the wrong way. You mean, that we will get only 4 records in the end...

**Review by YevgeniyYP — APPROVED** [review]

**Pull request merged**